schemas/vulnerability.yml

```diff
@@ -0,0 +1,144 @@
+---
+- name: vulnerability
+  title: Vulnerability
+  group: 2
+  short: Fields to describe the vulnerabilty relevant to an event.
+  description: >
+    The vulnerability fields describe information about a vulnerabilty that is relevant to an event.
+
+    Fields can have one entry or multiple entries.
+
+  reusable:
+    top_level: true
+    expected:
+      - client
+      - host
+      - os
+      - observer
+  type: group
+  fields:
+
+    - name: cvss.score.base
```

Comment:
It might be nice to version this since CVSS scores can vary dramatically depending on if v2 v. v3 calculations are used.

Reply:
thats probably a good idea (at least for the base fields - iirc theres a bunch of differences)

Reply:
There are differences in the scoring between versions, but both include a Base, Temporal, and Environmental calculation.

I have added `cvss.score.version` and will update the PR.